src/controllers/user.js

```diff
@@ -57,11 +57,57 @@ export const getAll = async (req, res) => {
 }
 
 export const updateById = async (req, res) => {
-  const { cohort_id: cohortId } = req.body
+  const id = Number(req.params.id)
+  const { firstName, lastName, bio, githubUrl, cohortId, profilePicture } =
+    req.body
 
-  if (!cohortId) {
-    return sendDataResponse(res, 400, { cohort_id: 'Cohort ID is required' })
-  }
+  try {
+    // Check user you want to update exists:
+    const foundUser = await User.findById(id)
```

Comment:
i think this should be `await dbClient.User.findById(id)` with `dbClient` imported from utils folder; this to ensure only one prisma connection is being created (as far as I know)